apps/admin/backend/src/tabulation/full_results.ts

```diff
@@ -57,6 +59,33 @@ export function tabulateCastVoteRecords({
   });
 }
 
+/**
+ * Memoized version of `tabulateCastVoteRecords`.
+ */
+const tabulateCastVoteRecordsMemoized = memoize(
```

Comment:
While I'm sure this works fine, do we actually need to memoize multiple values? Or do we just need to memoize _the most recent_ value?